instrumentation/spring/spring-pulsar-1.2/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,75 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("org.springframework.pulsar")
+    module.set("spring-pulsar")
+    versions.set("[1.2.0,)")
+  }
+}
+
+dependencies {
+  library("org.springframework.pulsar:spring-pulsar:1.2.0")
+  implementation(project(":instrumentation:pulsar:pulsar-2.8:javaagent"))
+
+  testInstrumentation(project(":instrumentation:pulsar:pulsar-2.8:javaagent"))
+
```

Comment:
```suggestion
```

Reply:
other instrumetnations also leave a blank line between dependencies in different scopes, here `testInstrumentation` and `testImplementation`

Reply:
> other instrumetnations also leave a blank line between dependencies in different scopes, here `testInstrumentation` and `testImplementation`

oops, I saw it wrong.